source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -3325,6 +3412,23 @@ class ActionCollection {
         this._owner = owner;
     }
 
+    parse(json: any, errors?: Array<IValidationError>) {        
```

Comment:
>         [](start = 56, length = 8)

looks like you haven't configured your editor to consume `.editorconfig` yet? instructions [here](https://editorconfig.org/#download).

Reply:
Done.